src/api/src/attestation/coco/grpc.rs

```diff
@@ -36,13 +36,26 @@ fn to_grpc_tee(tee: Tee) -> GrpcTee {
     }
 }
 
+#[derive(Clone, Debug, Deserialize)]
+pub struct GrpcConfig {
```

Comment:
Is there any value to defining all the different configs in the main `config.rs`?

Reply:
I'm not sure. It does make sense to me to let each implementation define which parameters it needs. In the same vein, `KbsConfig` isn't intended to be one central config struct which we pass around the entire codebase (I know we currently pass it to `AttestationService::new()` and am not fond of it but not sure if I have a better alternative).

But maybe there is merit to the idea of one central place for storing all config-related types.

WDYT?

P. S. As a side note to keep in mind, it's easy to run into circular dependencies here.

Reply:
>but not sure if I have a better alternative

Maybe we can have 3 constructors with feature toggles 👀

Reply:
Yeah I think it's fine either way tbh.

Reply:
OK, I've refactored so that we now explicitly pass an implementation-specific config to `AttestationService::new()` and use 3 implementations for this constructor.